File: MODUL03/PraktikumC/PRAK304-231081711004-AbdurrahmanGilangHarjuna.c
#include <stdio.h>

int main() {
    int a;
    scanf("%d", &a);

    if (a == 0) {
        printf("Nol");
    } else if (a >= 1 && a < 10) {
        printf("Satuan");
    } else if (a >=11 && a < 20) {
        printf("Belasan");
    } else if (a >= 20 && a < 100) {
        printf("Puluhan");
    } else if (a >= 100) {
        printf("Anda Menginput Melebihi Limit Bilangan");
    }

    return 0;
}

File: MODUL03/PraktikumPy/PRAK304-231081711004-AbdurrahmanGilangHarjuna.py
a = int(input())

if a == 0:
    print("Nol")
elif 1 <= a < 10:
    print("Satuan")
elif 11 <= a < 20:
    print("Belasan")
elif 20 <= a < 100:
    print("Puluhan")
elif a >= 100:
    print("Anda Menginput Melebihi Limit Bilangan")
